include/boost/geometry/algorithms/detail/overlay/sort_by_side.hpp

```diff
@@ -325,7 +325,7 @@ public :
                 double
             >::type;
 
-        ct_type const tolerance = 1000000000;
+        static auto const tolerance = common_approximately_equals_epsilon<ct_type>::value();
```

Comment:
this is the actual change right?

Reply:
Indeed!

Reply:
I'll document the `100` later. It's not new, it's moved. 
It is recently (earlier this year) adapted from a larger value to this value, considering another issue and all current tests.
It is used for clustering turns, and now also for sort by side. In both cases the exact value is not that important, but it should not be 1 (for floating point comparisons) and not be too large (obviously). The tolerance which was now replaced was very large, I don't remember exactly why.

Reply:
See also #1108